docs/DeveloperGuide.md

```diff
@@ -1094,6 +1129,14 @@ Every user can...
 - View a price list comparison of all available shops near NUS
 - Set recurring transactions
 
+### Known JavaFX Bugs
+
+- Light scrollbars hard to see
+
+### Individual Contributions and Efforts
+
+- [Jia Hao](team/woojiahao.html): get total expenditure, budget management logic (budget models were created by Lip Wei and Jamie), continuous integration, PR reviews
```

Comment:
I think the efforts appendix refers to the overall holisitc efforts by the whole team in the implementation of UniCa$h, and challenges faced, so that the evaluators can have a sense of overall effort required for UniCa$h benchmarked against the effort that would have been required for AB-3

![image](https://github.com/AY2324S1-CS2103-T16-3/tp/assets/19762596/bb100655-c90c-43f2-9afd-9a0db98781c1)

![image](https://github.com/AY2324S1-CS2103-T16-3/tp/assets/19762596/936d0747-6b00-4d23-9271-b0ae13a3fa6b)

Reply:
Oop, I thought it was the individual efforts, will fix

Reply:
<img width="987" alt="Screenshot 2023-11-13 at 14 56 49" src="https://github.com/AY2324S1-CS2103-T16-3/tp/assets/29497029/ac909abf-1310-4962-9e25-b3223f4f71e2">

Fixed the section